pandas/tslib.pyx

```diff
@@ -977,6 +953,30 @@ cdef class _Timestamp(datetime):
         self._assert_tzawareness_compat(other)
         return _cmp_scalar(self.value, ots.value, op)
 
+    def __repr__(self):
```

Comment:
so moving this to ``_Timestamp`` worked, interesting. Should move any referenced functions as well ,e.g. ``._repr_base``. ``_date_repr`` & ``_time_repr`` for consistency.